tools/remove_grouping_from_report_results.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+require 'trollop'
```

Comment:
`require 'optparse'` for lyfe!!1!  :trollface:

Reply:
I'm waiting for @kbrock to rename trollor to optimal or whatever we decided.  🕐

Reply:
or trollop

Reply:
I always favor stdlib things when possible, so that was what my comment was all about.